### ISSUE TITLE
--sourcemap=false does nothing?

### ISSUE THREAD

**Issue by Pomax** [developit/microbundle#773]
I'm running the following command:
```
> npx microbundle -i ./src/react-app/ui.js -o ./public/preact-ui.js --format=esm --sourcemap=false 
```
and while the `--format` flag works (only the ESM version gets generated) the `--sourcemap` flag seems to do nothing: there's still a `preact-ui.esm.js.map` file getting generated, with a comment that points to that file in the `preact-ui.esm.js` file.

**Comment by developit** [comment]
Funky! Can you try `--no-sourcemap`?

**Comment by Pomax** [comment]
Yep, `--no-sourcemap` works a treat - I see it mentioned in the "examples" for the CLI use, but it looks like it's missing from the list of flags itself?

**Comment by developit** [comment]
It's sortof a quirk of the arg parser we're using. I'll probably just add a fix so that `--sourcemap false` works too.
